examples/wow/zone_features__spark_v2.py

```diff
@@ -0,0 +1,40 @@
+import pyspark.sql as ps
+from pyspark.sql import functions as sf
+from zone_features__common import (
+    darkshore_flag,
+    darkshore_likelihood,
+    durotar_flag,
+    durotar_likelihood,
+)
+
+from hamilton.plugins.h_spark import with_columns
+
+
+def spark_session() -> ps.SparkSession:
+    return ps.SparkSession.builder.master("local[1]").getOrCreate()
+
+
+def world_of_warcraft(spark_session: ps.SparkSession) -> ps.DataFrame:
+    return spark_session.read.parquet("data/wow.parquet")
+
+
+@with_columns(darkshore_flag, durotar_flag, columns_to_pass=["zone"])
+def with_flags(world_of_warcraft: ps.DataFrame, aggregation_level: str) -> ps.DataFrame:
```

Comment:
aggregation_level doesn't seem to be needed here?

Reply:
Correct, that's a mistake